src/Microsoft.Restier.EntityFramework/DbApi.cs

```diff
@@ -60,14 +60,13 @@ protected T DbContext
         [CLSCompliant(false)]
         protected override IServiceCollection ConfigureApi(IServiceCollection services)
         {
-            services = base.ConfigureApi(services)
+            return base.ConfigureApi(services)
                 .CutoffPrevious<IModelBuilder>(ModelProducer.Instance)
                 .CutoffPrevious<IModelMapper>(new ModelMapper(typeof(T)))
                 .CutoffPrevious<IQueryExpressionSourcer, QueryExpressionSourcer>()
-                .CutoffPrevious<IQueryExecutor>(QueryExecutor.Instance)
+                .ChainPrevious<IQueryExecutor, QueryExecutor>()
```

Comment:
Do you mean we may have more inner queryexecutor? I have impression this will be the most inner QueryExecutor? Or we want to have DefaultQueryExecutor as most inner query executor?